database/build.gradle

```diff
@@ -7,7 +7,7 @@ var sqliteVersion = "3.44.0.0"
 dependencies {
     implementation 'com.google.code.findbugs:jsr305:3.0.2'
     implementation "org.xerial:sqlite-jdbc:${sqliteVersion}"
-    implementation 'org.flywaydb:flyway-core:10.1.0'
+    implementation 'org.flywaydb:flyway-core:10.0.0'
```

Comment:
why a downgrade? typo?

Reply:
I'm not sure, was confused as well. Possibly happened during merge.